### PR TITLE
chore(release): add website deploy to master script

### PR DESCRIPTION
**Pull request by christianalfoni**
Makes master branch also run the website publishing script

### CODE REVIEW

File: scripts/release/index.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ cooker.cook('publish', [
       cook.pushTagToRemote,
       cook.createReleaseNotes(releaseNotesTemplate),
       cook.createGithubRelease,
+      cook.runNpmScript('deploy'),
```
Comment (Guria) [inline scripts/release/index.js]: Have you tested it? I am not sure this will work. At least website dependencies is not installed automatically

Comment (christianalfoni) [inline scripts/release/index.js]: I have no idea how it works. You did the script and @gaspard did the new setup stuff :) @gaspard , you have any comments here?

Comment (gaspard) [inline scripts/release/index.js]: I think we need to install the devDependencies required for all this "shx", "run-p", "run-s", "gh-pages". Should be tried locally first with "npm run deploy" (eventually disconnecting network to test this)...

We could list the devDeps in "website/package.json" and this should update the main package.json file...

Comment (christianalfoni) [inline scripts/release/index.js]: cc @Guria , got any comments on this? Not sure how far I will get offline and do not understand how this script ever worked without having the dependencies?

**Comment by gaspard** [comment]
OK, I updated the deps for this and it should work now.

**Pull request merged**